ceph-releases/luminous/ubuntu/16.04/daemon/clean_container.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+INITIAL_SIZE=$(du -sm / 2>/dev/null | awk {'print $1}')
+DEBUG=
+
+# Step 1
+# Let's remove easy stuff
+strip -s /usr/local/bin/{confd,forego,kubectl}
+strip -s /usr/bin/{crushtool,monmaptool,osdmaptool}
+rm -rf /usr/share/{doc,info,locale,man}/*
+rm -f /usr/bin/{etcd-tester,etcd-dump-logs}
+
+# Some logfiles are not empty, there is no need to keep them
+for logfile in $(find /var/log/ -type f); do
+    echo > $logfile
+done
+
+STEP1_SIZE=$(du -sm / 2>/dev/null | awk {'print $1}')
+REMOVED_STEP1_SIZE=$(($INITIAL_SIZE - $STEP1_SIZE))
+echo "Stripping process: Step 1 removed ${REMOVED_STEP1_SIZE}MB"
+
+# Step 2
+# Let's remove stuff than can be discussed
+
+# Let's strip the ceph libraries
+strip -s /usr/lib/ceph/erasure-code/*
```

Comment:
@liewegas thoughts?

Reply:
I think this should be dropped and that this is the real problem: https://github.com/ceph/ceph/pull/19870

Reply:
Thanks for taking it upstream. I keep this change as it could be useful on existing & older releases and it really save space. Once your patch will be in a release, that line will be useless but without any impact.

Reply:
We'll be able to backport the packaging change to luminous, which means this is temporary.. I'd lean towards avoiding adding it in the first place.

Reply:
very nice @liewegas